### DISCUSSION TITLE
How do I fix "Cannot destructure property 'guild' of 'message.member' as it is null" error?

### DISCUSSION THREAD

**Discussion by EBMOfficial** [discordjs/discord.js#5232] [Q&A] [answered]
I am trying to make this function where the user tells the bot what channel to set up the points channel in. The points channel is basically a channel where users would post attachments and get upvotes/downvotes by other users. Everything is working except this:
`client.on('message', async message => {
  if (!message.guild) return;
  if (!message.channel.type === 'dm') return;
   // Destructure the guild property from the member object
    const { guild } = message.member
    // Access the channel ID for this guild from the cache
    const channelId = getChannelId(guild.id)
      
  if (message.channel.id = channelId) {
      if (message.attachments.size > 0) {
          message.react('🔼')
          message.react('🔽')
        } else {
          message.delete()
      }
  }
  
});`

This function is where the bot checks if the channel is indeed the points channel. If it is the points channel, the bot would react, else it would not. Errors are occurring at `const { guild } = message.member`. I am aware that message.member is gonna be null in these conditions, but I am not aware of how I can fix this.

**Comment by Hepka** [comment, marked as answer]
Why import a guild from message.member when you can do it from a simple message (without .member)?
![image](https://user-images.githubusercontent.com/61389918/105401403-9b1d0f00-5c3f-11eb-9430-505e563e832f.png)

**Reply by EBMOfficial** [reply]
Hi Hepka, one of my friends informed me about the single "=" issue, and as of now I have corrected it. But still, thanks for your response!
